### ISSUE TITLE
How do I just use AuthMe to link MC UUID to Wordpress?

### ISSUE THREAD

**Issue by FrozenBeard** [AuthMe/AuthMeReloaded#1667]
I want to use only one feature and nothing else in AuthMe, I want players to be able to link their minecraft account to their wordpress account by using a command in-game -> wordpress. 

I dont want any silly password to login, no stopping permissions, it needs to be entirely optional for players to link their minecraft to their wordpress.

This looks like what CommunityBridge did, but its very outdated.

**Comment by sgdc3** [comment]
try setting register: force: false

**Comment by GraappRM** [comment]
Did you find a way to manage your problem ?
I am facing the same problem.

**Comment by sgdc3** [comment]
@GraappRM have you tried my solution?

**Comment by GraappRM** [comment]
Yes but with this solution we can create new account from Minecraft but if a player already has an account on the website how can he link it ?

**Comment by FrozenBeard** [comment]
im also with @GraappRM I have 4000+ registered users on the website that just need to link their minecraft account (Wordpress).

**Comment by sgdc3** [comment]
this requires at leas a wordpress plugin, this task can't be handled by authme itself

**Comment by FrozenBeard** [comment]
I can write PHP & have made my own Wordpress plugins, tell me how I could do it and I can look into it.

**Comment by sgdc3** [comment]
The issue is that associating an account after it was created in the wp db is a mess

**Comment by FrozenBeard** [comment]
Any luck with this? I would still like this to work.
